tests/adapters/switches/cached_switch_test.py

```diff
@@ -321,6 +321,22 @@ def test_set_interface_state_on(self):
             self.switch.get_interfaces(),
             is_([Interface('xe-1/0/2', shutdown=False)]))
 
+    def test_unset_interface_state(self):
+        self.real_switch_mock.should_receive("get_interfaces").once() \
+            .and_return([Interface('xe-1/0/2')])
+        self.switch.get_interfaces()
+
+        self.real_switch_mock.should_receive("unset_interface_state").once() \
+            .with_args('xe-1/0/2')
+
+        self.real_switch_mock.should_receive("get_interface").once() \
```

Comment:
Before this line there should be a self.real_switch.get_interface... that will NOT use the cache and refetch it